spec.html

```diff
@@ -12748,7 +12748,7 @@ <h1>
               1. If _Desc_ has a [[Set]] field and SameValue(_Desc_.[[Set]], _current_.[[Set]]) is *false*, return *false*.
             1. Else if _current_.[[Writable]] is *false*, then
               1. If _Desc_ has a [[Writable]] field and _Desc_.[[Writable]] is *true*, return *false*.
-              1. If _Desc_ has a [[Value]] field and SameValue(_Desc_.[[Value]], _current_.[[Value]]) is *false*, return *false*.
+              1. If _Desc_ has a [[Value]] field, return SameValue(_Desc_.[[Value]], _current_.[[Value]]).
```

Comment:
Maybe a note to discourage future would-be refactorings?

Reply:
Unless you have a concrete suggestion on hand, I'd rather not delay this indefinitely. We could always add the note in a follow-up, right?

Reply:
NOTE: The following is not equivalent to returning *false* when SameValue returns *false* in the case of NaN.

Reply:
Nah, I don't like that wording. I'd say something like "the following step must not fall through to the property configuration steps below, even when SameValue returns true, because NaN values may be distinguishable".

Reply:
```suggestion
              1. NOTE: SameValue returns true for *NaN* values which may be distinguishable by other means. Returning here ensures that any existing property of _O_ remains unmodified.
              1. If _Desc_ has a [[Value]] field, return SameValue(_Desc_.[[Value]], _current_.[[Value]]).
```

Reply:
That also WFM.

Reply:
`*true*` btw

Reply:
sure, wfm too.